src/Tribe/Plugins.php

```diff
@@ -131,5 +131,136 @@ public function get_list() {
 			return apply_filters( 'tribe_plugins_get_list', $this->tribe_plugins );
 		}
 
+		/**
+		 * Returns true if the plugin "Event Tickets" is active.
+		 * Only reliable after "plugins_loaded" has been called.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool
+		 */
+		public static function is_event_tickets_active() {
```

Comment:
I think the methods are a great addition, however I would go with a more dynamic approach in order to favor the fact that we can add more plugins externally using filters and such, instead of creating custom functions for each plugin I think we can create a single function:

```php
public static function is_active( $plugin_name ) {
	$plugins = apply_filters( 'tribe_active_plugins', [] );
	return isset( $plugins[ $plugin_name ] );
}
```

So this one can be registered from the plugin side such as TEC or ET.

```php
// from Event tickets
add_filter( 'tribe_active_plugins', static function( $plugins ) {
	$plugins[ 'et' ] = true;
	$plugins[ Tribe__Events__Main::class ] = true;
	$plugins[ 'event_tickets' ] = true;
	$plugins[ 'tickets' ] = true; 
});
```

This allow to use different alias for different plugins and use the function such as: 

```php
new ( Tribe__Plugins )->is_active( 'et' );
new ( Tribe__Plugins )->is_active( 'tickets' );
```

Another benefit of this approach is the removal of concerns between common and external plugins, so in this way the plugins always have the control on how to respond to the `is_active` function.

Reply:
Interesting. The plugin declares itself as loaded, through the filter... Pretty clever.

What do you think we wait a little bit to see if someone else has thoughts on it before I update this PR?

Reply:
I would love to see something like what @mitogh suggested there. But I would possibly throw a `_doing_it_wrong` if called before `plugins_loaded` which would ensure we avoid problems in the fugure.

Reply:
Can we stick to one plugin key for each plugin instead of all of the aliases (the repo slug)?

* event-tickets
* event-tickets-plus
* the-events-calendar
* events-pro

etc..

Reply:
I think using the same slug from the containers would be ideal: 

- https://moderntribe.github.io/products-engineering/docs/plugins/namespacing/

To keep it consistent with the rest of containers where a plugin main container is registered.

Reply:
Container slugs is perfect here!